include/dt-bindings/gpio/gpio.h

```diff
@@ -102,6 +102,59 @@
 
 /** @} */
 
+/**
```

Comment:
Add comment about these being zephyr specific like the voltages are.

Reply:
Added, thanks.